policy.go

```diff
@@ -26,11 +26,10 @@ const (
 func calcMinRequiredTxRelayFee(serializedSize int64, minRelayTxFee btcutil.Amount) int64 {
 	// Calculate the minimum fee for a transaction to be allowed into the
 	// mempool and relayed by scaling the base fee (which is the minimum
-	// free transaction relay fee).  minRelayTxFee is in Satoshi/KB, so
-	// divide the transaction size by 1000 to convert to kilobytes.  Also,
-	// integer division is used so fees only increase on full kilobyte
-	// boundaries.
-	minFee := (1 + serializedSize/1000) * int64(minRelayTxFee)
+	// free transaction relay fee). minTxRelayFee is in Satoshi/kB so
+	// multiply by serializedSize (which is in bytes) and divide by 1000 to get
+	// minimum Satoshis.
+	minFee := (serializedSize * int64(minRelayTxFee)) / 1000
 
```

Comment:
There is a case missing here (I had it in my original comment):

```Go
if minFee == 0 && minRelayTxFee > 0 {
    minFee = minRelayTxFee
}
```

This is because there are 2 cases in which the `minFee` can be zero:
1) A zero serializedSize (which is actually impossible in practice)
2) When serializedSize * minRelayTxFee < 1000 (due to integer division)

The additional if statement covers the second case to ensure a combination of the serialized tx size and relay rates that would result in values too small will not result in a 0 fee.